examples/react-native/v12/TestApp/src/components/subscribeApi/WaitFirstTime.tsx

```diff
@@ -0,0 +1,102 @@
+// :snippet-start: wait-firsttime
+import React, {useEffect, useState} from 'react';
+import {BSON, WaitForSync} from 'realm';
+import {useRealm, useQuery} from '@realm/react';
+import {View, Text, Button, TextInput, FlatList} from 'react-native';
+
+import {Bird} from './models/Bird';
+import {Subscription} from 'realm/dist/bundle';
+
+export const WaitFirstTime = () => {
+  const realm = useRealm();
+  const [birdName, setBirdName] = useState('Change me!');
+
+  // Get local birds that have been marked as "haveSeen".
+  const seenBirds = useQuery(Bird).filtered('haveSeen == true'); // :emphasize:
```

Comment:
```suggestion
  const seenBirds = useQuery(Bird, col => col.filtered('haveSeen == true')); // :emphasize:
```
Let's make this syntax standard.  Otherwise on every re-render the collection reference will be updated.

Reply:
Agh, right. Thanks for reminding me. I need to break the old habit.